appveyor.yml

```diff
@@ -5,7 +5,6 @@ init:
 install:
   - SET PATH=C:\Ruby21-x64\bin;%PATH%
   - SET LOG_SPEC_ORDER=true
-  - gem install bundler --quiet --no-ri --no-rdoc
```

Comment:
Puppet is probably fine to drop this line though. We may still need it in modules.